src/app/docs/button/button-docs.component.html

```diff
@@ -197,7 +229,7 @@
             Use singular or plural direct objects to indicate whether the button acts on one or multiple items.
           </li>
           <li>
-            Don't include articles. For example, use <strong>Add constituent</strong> rather than <strong>Add a constituent</strong>.
+            Don't include articles. For example, use <strong>Delete constituent</strong> rather than <strong>Delete a constituent</strong>.
```

Comment:
```suggestion
            Don't include articles. For example, use <strong>Delete goal</strong> rather than <strong>Delete a goal</strong>.
```
It occurs to me that given we are tweaking these docs to correctly reflect the add vs. new terminology for button labels, maybe our examples shouldn't include "constituent" when we are moving away from that term in preference of "contact." I'd suggest something more neutral that won't eventually be outdated.

Reply:
@blackbaud-johnly Great catch on the article guidelines with constituents still referenced. thx.